modules/foxglove_websocket/1.3.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "foxglove_websocket",
+    srcs = glob(["cpp/foxglove-websocket/src/*.cpp"]),
+    hdrs = glob(["cpp/foxglove-websocket/include/foxglove/websocket/*.hpp"]),
+    copts = ["-std=c++17"],
```

Comment:
Could this be moved to the build flags? It wouldn't necessarily be safe to mix standards in a single build.

Reply:
I've moved it to presubmit.yml